ajax/libs/material-scrolltop/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "material-scrolltop",
+  "filename": "material-scrolltop.min.js",
+  "version": "1.0.0",
+  "description": "Lightweight, Material Design inspired plugin for scrolling on top of the html page (with jQuery)",
+  "author": "Lukas Bartak <bart@bartweb.cz> (https://github.com/bartholomej)",
+  "repository": {
+    "url": "git+https://github.com/bartholomej/material-scrolltop.git",
+    "type": "git"
+  },
+  "keywords": [
+    "material",
+    "material design",
+    "ripple",
+    "button",
+    "animate",
+    "jquery",
+    "scrollTop",
+    "scroll to top",
+    "sass"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bartholomej/material-scrolltop.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "material-scrolltop*.+(css|js)",
```

Comment:
There might be `.map` files in the future. So let's update this to be `material-scrolltop*.+(css|js|map)`